disnake/utils.py

```diff
@@ -1369,3 +1369,9 @@ def _overload_with_permissions(func: T) -> T:
 # this is used as a marker for functions or classes that were created by codemodding
 def _generated(func: T) -> T:
     return func
+
+
+# Similar to typing.assert_never, but returns instead of raising (i.e. has no runtime effect).
+# This is only to avoid "unreachable code", which pyright doesn't type-check.
+def assert_never(arg: Never, /) -> None:
+    pass
```

Comment:
could we possibly move this above the _generated marker?

Reply:
Any particular reason for that? Should add a note to the file in that case